### ISSUE TITLE
Create user onboarding tutorial

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#130]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

Initial user tests have indicated that a user tutorial would be a useful addition to the app screen. The general plan for this will be another partnership with Spencer who did the app icon and logo. He's going to be making a workflow that can then be swiped through by users. Resources related to swiping include [this](https://stackoverflow.com/questions/24215117/how-to-recognize-swipe-in-all-4-directions), [this](https://stackoverflow.com/questions/30875072/change-ios-view-view-controller-using-swipe-gesture) and [this](https://cocoacasts.com/swift-fundamentals-working-with-swipe-gesture-recognizers-in-swift), with the Apple documentation for it being found [here](https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_uikit_gestures/handling_swipe_gestures).

- This issue is being done in tandem with #16, as the plan is to move the privacy policy button to a hamburger menu at the top right and then have that button instead be a "Quick Tutorial" button. This button will trigger the workflow.

### Contribution

I'll be doing the codes for this, but help is welcome :)

**Comment by andrewtavis** [comment]
arrow.left.arrow.right (SF Symbols) and each of those individually can be used to indicate to the user that a swipe is possible.

**Comment by wkyoshida** [comment]
Leaving some notes here that started in [this discussion](https://github.com/scribe-org/Scribe-iOS/issues/229#issuecomment-1284356978): 

> Scribe could potentially just quickly prompt the user for the currency symbol during a keyboard setup - if keyboard selection is implemented as such
...
The `Home` page could have a `Set my keyboard up` form-like walkthrough with the options, target language, source language, keyboard layout, currency, etc.

I wasn't sure at first what the onboarding would end up showcasing in reality, but @andrewtavis mentioned the above would apply here! :+1: 

Also, I'm not sure what else has already been proposed to showcase in the onboarding, but some other ideas could be:
- Quick note on what the abbreviations in the keyboard mean, eg.:
  - `M` = masculine
  - `F` = feminine
  - `Akk` = in the German keyboard, accusative
  - etc.
  - Though, would have to consider how to do this per language, as the abbreviations would be different. Perhaps a page per language with a quick list.

**Comment by andrewtavis** [comment]
Yes we'd likely have to ask them what language they want to have the tutorial in, @wkyoshida 🤔 Definitely the above would be included, and it'd be great to remove the preposition-case relationships in case the language they select doesn't include them. Also putting in translation and conjugation would be good, as well as a reference to them changing things in settings.

The abbreviations are already translated into the keyboard language via the following:
https://github.com/scribe-org/Scribe-iOS/blob/3fbd32a39120c6fa39aa7e09f6a503a3bf6cc3c7/Keyboards/KeyboardsBase/ScribeFunctionality/Annotate.swift#L25

We'd maybe need to move it into a more central place, but should be easy enough to store them in one place 😊

Thanks for moving all this over here, btw! 🚀🙏

**Comment by andrewtavis** [comment]
And just FYI, @wkyoshida, I'm slammed for another project through Thursday, but do have all this in mind :) Will also get some feedback from a senior iOS dev about how to do the database #89 hopefully by the end of the week 😊
